module3/projects/take_home/tea_take_home_fe.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Tea Subscription UI
+subheading: Take-Home Front End Prompt
+layout: page
+---
+
+## Things we'll be looking for:
+
+* A strong understanding of React JS and Front-End best practices
+* Your ability to prioritize for MVP
+* An understanding of basic usability practices and standards
+* A clear hierarchy of information
+* Clean, well thought out code
+
+Try to limit your total time to around 8 hours on this portion of the project. Prioritize what is important to get this working and as close to MVP as possible. Please do not attempt to recreate an entire existing Tea Shop's website. We're interested in seeing your ability to write clean code to solve a problem with an understanding of basic UX. We're not looking for fancy UI tricks or hacks.
+
+## Tea Subscription
+
+You will create a tea subscription application that...
+
+### Requirements:
+
+At a minimum, you must show:
+
+* A list of teas (name, image, description if present)
+* A "detailed" view for each tea (name, image, price, description, subscription type, and button to subscribe)
+  * Teas in the list must link to a detailed tea view within your app (linking to the source website does not fulfill this requirement)
+* Some sort of search, filter, OR sort
```

Comment:
I'd love to get others' input on this, but I wonder if we can make this a bit more open ended to allow different types of implementations, and allow for more problem-solving about how to communicate with the BE and how to structure the data in the db. Something like:
1) List of teas (name, image)
2) Detailed View (name, image, price, description)
3) The ability to create a tea subscription (for one tea at a time or multiple teas) somewhere in the application

Does this seem reasonable for students to do in 8 hours?

Reply:
Yes, absolutely reasonable for 8 hours. Honestly in 8 hours focused only on frontend, I would expect some testing too